src/Tests/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs

```diff
@@ -17,8 +18,24 @@ public class WorkloadSetTests : VMTestBase
     {
         readonly string SdkTestingDirectory = @"C:\SdkTesting";
 
+
+        Lazy<Dictionary<string, string>> _testWorkloadSetVersions;
+        string WorkloadSetVersion1 => _testWorkloadSetVersions.Value.GetValueOrDefault("version1", "8.0.300-preview.0.24178.1");
+        string WorkloadSetVersion2 => _testWorkloadSetVersions.Value.GetValueOrDefault("version2", "8.0.300-preview.0.24217.2");
+        string WorkloadSetPreviousBandVersion => _testWorkloadSetVersions.Value.GetValueOrDefault("previousbandversion", "8.0.204");
+
         public WorkloadSetTests(ITestOutputHelper log) : base(log)
         {
+            _testWorkloadSetVersions = new Lazy<Dictionary<string, string>>(() =>
+            {
+                var versionsFile = VM.GetRemoteFile(@"c:\SdkTesting\workloadsets\testworkloadsetversions.json");
```

Comment:
Should this file be documented at https://github.com/dotnet/sdk/blob/main/test/dotnet-MsiInstallation.Tests/README.md ?